### PR TITLE
feat: add contribution icon in homepage

### LINKED ISSUE #642: Add kontribusi icon on homepage
## Description

Screenshot_20210811-205227_Chrome

Link goes to: https://www.indorelawan.org/activity/60e2ed45164da80018b0e246

## Figma

https://www.figma.com/file/XNNtIoFEdFqaXOee83n0oN/WBW?node-id=485:2485

## Tasks

- [ ] Add new icon on homepage
- [ ] Add testing

### PR DESCRIPTION
**Pull request by hariangr**
Closes #642  

## Description

This pull request add a new icon to homepage (Kontribusi) with dynamic property (fill) and a test
image

## Current Tasks

- [x] Add new icon on homepage
- [x] Add testing

### CODE REVIEW

File: lib/home/homepage-menus.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import {
   OxygenIcon,
   DonationIcon,
 } from "~/components/ui/icons";
+import { ContributionIcon } from "~/components/ui/icons/contribution-icon";
```
Comment (resir014) [inline lib/home/homepage-menus.tsx:11]:
```suggestion
  DonationIcon,
  ContributionIcon,
} from "~/components/ui/icons";
```

Please include this icon inside the grouped exports on the `~/components/ui/icons/index.ts` file.

**Review by resir014 — CHANGES_REQUESTED** [review]
Some code structure issues. Please look into them! @hariangr

**Comment by resir014** [comment]
We should also add an `external` option to the `HomepageMenuItem` interface, to make sure external links get properly handled when this option is set (e.g. `target="_blank"` and proper `rel` attributes).

I think we can do this on a separate PR. What do y'all think? @mazipan @zainfathoni

**Review by mazipan — APPROVED** [review]
Looks good to me

**Pull request merged**